app/gui/view/documentation/src/html.rs

```diff
@@ -122,23 +148,20 @@ fn render_virtual_component_group_docs(name: ImString) -> String {
 /// - Synopsis and a list of constructors.
 /// - Methods.
 /// - Examples.
-fn render_type_documentation(
-    docs: &TypeDocumentation,
-    function_name: Option<&QualifiedName>,
-) -> String {
+fn render_type_documentation(docs: &TypeDocumentation, back_link: Option<BackLink>) -> String {
     let methods_exist = !docs.methods.is_empty();
     let examples_exist = !docs.examples.is_empty();
     let name = &docs.name;
     let arguments = &docs.arguments;
     let synopsis = &docs.synopsis;
     let constructors = &docs.constructors;
-    let synopsis = section_content(type_synopsis(synopsis, constructors, function_name));
-    let methods = section_content(list_of_functions(&docs.methods, function_name));
+    let synopsis = section_content(type_synopsis(synopsis, constructors));
+    let methods = section_content(list_of_functions(&docs.methods));
     let examples = section_content(list_of_examples(&docs.examples));
     let tags = section_content(list_of_tags(&docs.tags));
 
     let content = owned_html! {
-        : header(ICON_TYPE, type_header(name.name(), arguments_list(arguments)));
+        : header(ICON_TYPE, type_header(name.name(), arguments_list(arguments), back_link.clone()));
```

Comment:
Do we need this clone? If yes, the method should take argument by reference, IMO.

Reply:
Changed it to the reference. I'm still not quite following the borrowing rules of horrorshow (of `owned_html!` macro).